README.md

```diff
@@ -7,7 +7,7 @@
 These are set in the your enviromental variables. These must be set before you can run the app
 
 ```
-NODE_ENV = Eviroment name ("development", "testing", "production")
+NODE_ENV = Eviroment name ("development", "test", "production")
```

Comment:
this was to get jest to work properly

Reply:
While you're at it, can we spell `Environment` right? lol
```suggestion
NODE_ENV = Enviroment name ("development", "test", "production")
```